include/onnxruntime/core/graph/graph.h

```diff
@@ -66,13 +67,18 @@ class Node {
     /** Gets the Node that this EdgeEnd refers to. */
     const Node& GetNode() const noexcept;
 
-    /** Gets the NodeArg that this edge end refers to.
-    @returns NodeArg pointer or nullptr if this is a control edge. */
-    const NodeArg* GetNodeArg() const noexcept;
+    /** Gets the source slot index.
+	@returns the source slot index of <*this> edge.*/
+    int GetSourceSlotIndex() const;
```

Comment:
Is it better to change it to GetSrcSlotIndex to align with GetDstSlotIndex below?

Reply:
We probably should just call it GetSrcNodeArgIndex and GetDstNodeArgIndex?

Reply:
Sure. Thanks!